deployment/factorycube-server/values.yaml

```diff
@@ -5,7 +5,10 @@
 ### factoryinsight ###
 factoryinsight:
   enabled: true
-  image: unitedmanufacturinghub/factoryinsight
+  image:
```

Comment:
Please use the unitedmanufacturinghub repo

Reply:
Done